src/Groups/GAPGroups.jl

```diff
@@ -323,22 +329,22 @@ julia> cperm([1,2],[2,3])
 (1,3,2)
 ```
 """
-function cperm(L::AbstractVector{<:Base.Integer}...)
+function cperm(L::AbstractVector{T}...) where T
    if length(L)==0
       return one(symmetric_group(1))
    else
-      return prod([PermGroupElem(symmetric_group(maximum(y)), GAP.Globals.CycleFromList(GAP.julia_to_gap(collect(y)))) for y in L])
+      return prod([PermGroupElem(symmetric_group(maximum(y)), GAP.Globals.CycleFromList(GAP.julia_to_gap(collect([Int(k) for k in y])))) for y in L])
    end
 end
 
 # cperm stays for "cycle permutation", but we can change name if we want
 # takes as input a list of arrays (not necessarly disjoint)
 # WARNING: we allow e.g. PermList([2,3,1,4,5,6]) in Sym(3)
-function cperm(g::PermGroup,L::AbstractVector{<:Base.Integer}...)
+function cperm(g::PermGroup,L::AbstractVector{T}...) where T
    if length(L)==0
       return one(g)
    else
-      x=GAP.Globals.Product(GAP.julia_to_gap([GAP.Globals.CycleFromList(GAP.julia_to_gap(collect(y))) for y in L]))
+      x=GAP.Globals.Product(GAP.julia_to_gap([GAP.Globals.CycleFromList(GAP.julia_to_gap(collect([Int(k) for k in collect(y)]))) for y in L]))
```

Comment:
Why use collect like this (see above)?

Reply:
How about this (untested):
```suggestion
      x=prod(y -> GAP.Globals.CycleFromList(GAP.julia_to_gap(collect(Int, y))), L)
```

Reply:
It works. I replaced it. I did not make a similar replacement for line 336 (the function above) because it needs to keep track of the degree of the permutation.